include/vsg/vk/Descriptor.h

```diff
@@ -160,7 +160,7 @@ namespace vsg
             }
         }
 
-        virtual bool assignTo(VkWriteDescriptorSet& wds, VkDescriptorSet descriptorSet) const
+        virtual bool assignTo(VkWriteDescriptorSet& wds, VkDescriptorSet descriptorSet) const override
```

Comment:
Good catch, though you forgot to remove the virtual ;-)

I'll do this once I've merged this PR.